kolibri/plugins/setup_wizard/assets/src/actions.js

```diff
@@ -0,0 +1,24 @@
+const Kolibri = require('kolibri');
+
+const DeviceOwnerResource = Kolibri.resources.DeviceOwnerResource;
+const FacilityResource = Kolibri.resources.FacilityResource;
+
+function createDeviceOwnerAndFacility(store, deviceownerpayload, facilitypayload) {
+  const DeviceOwnerModel = DeviceOwnerResource.createModel(deviceownerpayload);
+  const deviceOwnerPromise = DeviceOwnerModel.save(deviceownerpayload);
```

Comment:
both createModel and save methods require the payload parameter, otherwise error will throw.